apps/mobile/.env.production

```
@@ -0,0 +1,25 @@
+# Production Environment Variables
+NODE_ENV=production
+API_BASE_URL=https://api.wheresballthough.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_BASE_URL key should go before the NODE_ENV key

(UnorderedKey)